### DISCUSSION TITLE
No relevant ecu file found

### DISCUSSION THREAD

**Discussion by chiptuner123** [cedricp/ddt4all#1329] [Q&A]
Hello I do full scan and i have found ECUs like Red collor and Green collor.What this message mean? (No relevant ecu file founNo relevant ecu file found)  I use last version 3.0.3 and last database 6000 ecus.
How can fix this red ecus?
![11](https://github.com/user-attachments/assets/a04c37d2-cdda-41e9-a548-3cc312adbfbf)

Sorry for my question I am new user of this great tool,Also where can I have any support?

**Comment by Furtif** [comment]
🤔

**Reply by chiptuner123** [reply]
this car Koleos II ,are in data base but many ecus not regognized,like ABS vdc that i need to configurate the roller activation. No relevant ecu file founNo relevant ecu file found

**Comment by AndyIreland** [comment]
try selecting manually - 

If 'read' parameters are in Red colour(color) with 'No Data' in the 'READ' parameter boxes then its more than likely wrong ecu selected 

try one od these:
![image](https://github.com/user-attachments/assets/c5451b24-d50a-4cef-b693-30de4d909d67)

**Comment by chiptuner123** [comment]
Thank you I try all of them but not any one work. I have 6000 ecus database is the last update? you know?

**Reply by AndyIreland** [reply]
are you using a correct ELM327 with pic18f25k80 chip in it or cheap chines clone ELM327 - is your car a later one with security gateway locked. Try changing CAN timeout to 600ms and refresh

**Reply by chiptuner123** [reply]
I use original obd link Usb cable

**Reply by Furtif** [reply]
remove you DB and put one DB ok https://github.com/cedricp/ddt4all/discussions/912

**Reply by Furtif** [reply]
and Scan CAN

**Reply by chiptuner123** [reply]
I use this Ok i will try.Thanks to all for help!
